packages/amplify-e2e-core/src/utils/sdk-calls.ts

```
@@ -144,6 +144,32 @@ export const getUserPoolClients = async (userPoolId: string, clientIds: string[]
   return res;
 };
 
+export const addUserToUserPool = async (userPoolId: string, region: string) => {
+  const provider = new CognitoIdentityServiceProvider({ region });
+  try {
+    const params = {
+      UserPoolId: userPoolId /* required */,
+      Username: 'testUser' /* required */,
+      UserAttributes: [
+        {
+          Name: 'demoKey' /* required */,
+          Value: 'randomValue',
+        },
+        /* more items */
+      ],
+      ValidationData: [
+        {
+          Name: 'demoKey' /* required */,
+          Value: 'demoValue',
+        },
+      ],
+    };
+    const res = await provider.adminCreateUser(params);
+  } catch (e) {
+    console.log(e);
```

Why don't we want to surface the error here in our tests?